charts/redis-ha/templates/_configs.tpl

```diff
@@ -501,7 +501,7 @@
       {{- end}}
         ping
     )
-    if [ "$response" != "PONG" ]; then
+    if [ "$response" != "PONG" ] && [ "${response:0:7}" != "LOADING" ] ; then
```

Comment:
I'm confused on how this is supposed to work. We're saying in your code; if it's not PONG AND not LOADING then exit with error code 1. 

Isn't the objective to basically say if we receive something OTHER than PONG, exit and fail? Which is what the original is doing?

Reply:
During sync, the response is `LOADING: ...`, if we `exit 1` during sync the container will be restarted

| response        | liveness           |
| ------------- |:-------------:|
| PONG      | ok |
| LOADING      | ok      |
| .* | not ok      |